### ISSUE TITLE
0.16 Image does not appear to be available

### ISSUE THREAD

**Issue by garethjevans** [brancz/kube-rbac-proxy#280]
Hi, we're keen to upgrade to the latest (0.16) release, but can't seem to find the image location?

**Comment by ibihim** [comment]
@garethjevans, could you be more specific?

I checked the image page: https://quay.io/repository/brancz/kube-rbac-proxy?tab=tags, and it shows v0.16.0.

I also pulled it successfully:

```bash
$ podman image pull quay.io/brancz/kube-rbac-proxy:v0.16.0
Trying to pull quay.io/brancz/kube-rbac-proxy:v0.16.0...
Getting image source signatures
Copying blob sha256:d3c894b5b2b0fa857549aeb6cbc38b038b5b2828736be37b6d9fff0b886f12fd
Copying blob sha256:6ec9caf47f8f176a2299fbfc83c403d27535e8937c9bcca775308eb2ae467486
Copying blob sha256:960043b8858c3c30f1d79dcc49adb2804fd35c2510729e67685b298b2ca746b7
Copying blob sha256:fe09e3ee53d23c69e397e65622c72a24d06d07a3906237caa7bbcdc4614a3437
Copying blob sha256:eebb06941f3e57b2e40a0e9cbd798dacef9b04d89ebaa8896be5f17c976f8666
Copying blob sha256:02cd68c0cbf64abe9738767877756b33f50fff5d88583fdc74b66beffa77694b
Copying blob sha256:b40161cd83fc5d470d6abe50e87aa288481b6b89137012881d74187cfbf9f502
Copying blob sha256:46ba3f23f1d3fb1440deeb279716e4377e79e61736ec2227270349b9618a0fdd
Copying blob sha256:4fa131a1b726b2d6468d461e7d8867a2157d5671f712461d8abd126155fdf9ce
Copying blob sha256:5655aa5aaa2b9661e5a66c4aeca459ccacf54177aef014ea3f45cd29645a47fb
Copying config sha256:2e4f0cff00eb27ccf559d9e80b7f4f46c673dcab0979aa1838718df415d4c1ee
Writing manifest to image destination
2e4f0cff00eb27ccf559d9e80b7f4f46c673dcab0979aa1838718df415d4c1ee
```

**Comment by garethjevans** [comment]
Ah, thanks, I was looking at gcr.io/kubebuilder/kube-rbac-proxy

**Comment by mathieu-benoit** [comment]
So just to confirm, @ibihim, so from 0.16.0, this images won't be available anymore in gcr.io/kubebuilder/kube-rbac-proxy?

**Comment by JoaoBraveCoding** [comment]
For a detailed explanation on why there are no more updates to `gcr.io/kubebuilder/kube-rbac-proxy` see https://github.com/kubernetes-sigs/kubebuilder/discussions/3907

TL;DR: if you really want to use kube-rbac-proxy then you should use [quay.io/brancz/kube-rbac-proxy](https://quay.io/repository/brancz/kube-rbac-proxy)
